cypress/fixtures/users.json

```
@@ -0,0 +1,38 @@
+[
+  {
+    "username": "karadmin",
+    "rolename": "DistrictAdmin"
+  },
+  {
+    "username": "karadminro",
+    "rolename": "DistrictReadOnlyAdmin"
+  },
+  {
+    "username": "karstaff",
```

```suggestion
    "username": "karstaff1",
```
It seems `karstaff` user has been deleted so added new user